macros.in

```diff
@@ -195,11 +198,12 @@ package or when debugging this package.\
 %{nil}
 
 %debug_package \
-%ifnarch noarch\
 %global __debug_package 1\
+%(cat > "%{specpartsdir}/rpm-debuginfo.specpart" << EOL\
```

Comment:
I'd like it a whole lot more if the `cat` was in the template part. This is an ugly side-effect for a macro to have, eg somebody doing `rpm --eval "%{debug_package}"` to see what it's made of.